docs/project_setup.md

```diff
@@ -0,0 +1,50 @@
+## Step 1: Basic Project Setup
+
+A great guide to starting off the project can be found [here](http://www.agileventures.org/articles/project-setup-new-users) (use https://github.com/AgileVentures/WebsiteOne as the project URL).
+
+## Step 2: Run the setup script
+
+Get the project dependencies by running these 2 commands in a bash shell terminal
+
+    wget https://github.com/AgileVentures/setup-scripts/raw/develop/scripts/rails_setup.sh
+    WITH_PHANTOMJS=true REQUIRED_RUBY=2.2.2  source rails_setup.sh
+
+This script works best with Ubuntu 14.04 (Trusty Tahr) and Mac OS X 10.9 Mavericks, but please contribute installation instructions for other platforms.
+
+**Note:** On OSX El Capitan and above, you may get this error:
```

Comment:
I notice we have a reference to troubleshooting at the bottom; should notes like that be added in there?